Application/EdFi.Ods.Standard/Artifacts/Metadata/ApiModel.json

```diff
@@ -3,7 +3,7 @@
   "schemaDefinition": {
     "logicalName": "Ed-Fi",
     "physicalName": "edfi",
-    "version": "3.3.0-b"
+    "version": "3.3.0-a"
```

Comment:
The rebase on this seems to have reverted a bunch of data standard changes, this file and all the xsds and however many more files.